tracing-subscriber/src/fmt/mod.rs

```diff
@@ -608,18 +579,6 @@ impl<N, E, F, W> SubscriberBuilder<N, E, F, W> {
         }
     }
 
-    /// Sets the function that the subscriber being built should use to format
-    /// events that occur.
-    #[deprecated(since = "0.2.0", note = "renamed to `event_format`.")]
-    pub fn on_event<E2>(self, fmt_event: E2) -> SubscriberBuilder<N, E2, F, W>
-    where
-        E2: FormatEvent<Registry, N> + 'static,
-        N: for<'writer> FormatFields<'writer> + 'static,
-        W: MakeWriter + 'static,
-    {
-        self.event_format(fmt_event)
-    }
-
```

Comment:
this should not have been removed; it was renamed in 0.2.0 so there hasn't yet been a published version where this method was deprecated

Reply:
ah, fixed. thanks!